src/items/extern-crates.md

```diff
@@ -2,11 +2,12 @@
 
 > **<sup>Syntax:<sup>**\
 > _ExternCrate_ :\
-> &nbsp;&nbsp; `extern` `crate` [IDENTIFIER]&nbsp;(`as` ( [IDENTIFIER] | `_` ) )<sup>?</sup> `;`
+> &nbsp;&nbsp; `extern` `crate` ( [IDENTIFIER] | `self` ) (`as` ( [IDENTIFIER] | `_` ) )<sup>?</sup> `;`
```

Comment:
Since you've referenced the "as clause" below, I'd pull that bit out to a separate production:
```
AsClause :
  `as` ( [IDENTIFIER] | `_` )
```
I'd also pull `( [IDENTIFIER] | self )` out into a production `CrateRef`.